tensorflow/compiler/xla/service/cpu/BUILD

```diff
@@ -295,6 +295,7 @@ cc_library(
         "@llvm//:object",
         "@llvm//:support",
         "@llvm//:target",
+        "@llvm//:polly",
```

Comment:
@sanjoy this is a very good comment. I think having experimental optimizations, such as in LLVM having we have experimental backends is a very good idea. Polly certainly is not a -enabled-and-supported-by-default feature.